src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionAnnotatedTypeFactory.java

```diff
@@ -8,26 +8,36 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
The default logging implementation in the JDK has some issues and is not widely used. It's probably fine for now (since we're not deploying this in production or anything), but in general it's good to know not to use it.

In my experience, the most popular logging tooling in Java is SLF4J + Logback, but I think I'm a year or two out of date on best practices there.